neat-form-core/src/main/java/com/nerdstone/neatformcore/views/widgets/CheckBoxNFormView.kt

```diff
@@ -69,4 +77,39 @@ class CheckBoxNFormView : CheckBox, NFormView {
         super.setVisibility(visibility)
         visibilityChangeListener?.onVisibilityChanged(this, visibility)
     }
+
+
+    /**
+     * Obtain custom xml attributes passed for the stepper view
+     */
+    private fun setupViewAttributes(attrs: AttributeSet?) {
+        if (attrs != null) {
+            val typedArray = context.obtainStyledAttributes(
+                attrs, R.styleable.CheckBoxNFormView, 0, 0
+            )
+            try {
```

Comment:
Are we catching any exeption with this try block? If not then I don't think its needed.